src/components/App/App.js

```diff
@@ -2,23 +2,23 @@ import React from 'react'
 import logo from './logo.svg'
```

Comment:
It's recommended to use `index.js` as the name of the main file inside a folder. This allows you to import the component easily like this: `import App from 'components/App'`.